src/core/border_router/routing_manager.cpp

```diff
@@ -1019,6 +1149,24 @@ void RoutingManager::HandleRoutingPolicyTimer(Timer &aTimer)
     aTimer.Get<RoutingManager>().EvaluateRoutingPolicy();
 }
 
+#if OPENTHREAD_CONFIG_BORDER_ROUTING_NAT64_ENABLE
+void RoutingManager::HandleInfraIfNat64PrefixStaleTimer(Timer &aTimer)
+{
+    aTimer.Get<RoutingManager>().HandleInfraIfNat64PrefixStaleTimer();
+}
+
+void RoutingManager::HandleInfraIfNat64PrefixStaleTimer(void)
+{
+    RequestInfraIfNat64Prefix();
+
+    if (!mInfraIfNat64PrefixStaleTimer.IsRunning())
```

Comment:
`mInfraIfNat64PrefixStaleTimer` timer must not be running in the timeout handler?